manifests/python.yml

```diff
@@ -6,6 +6,13 @@ tests/:
       Test_SingleSpan: missing_feature (missing /e2e_otel_span endpoint on weblog)
   appsec/:
     api_security/:
+      test_apisec_sampling.py:
+        Test_API_Security_sampling:
+          '*': missing_feature
+          django-poc: v2.5.0
+          fastapi: v2.5.0
+          flask-poc: v2.5.0
```

Comment:
The uwsgi-poc variant is based in the flask one, it's not working on this one ? If it does, you can simply `Test_API_Security_sampling: v2.5.0`

Reply:
I'll check that, thanks!